src/main/java/com/hltech/pact/gen/domain/pact/PactFactory.java

```diff
@@ -40,26 +40,39 @@ public Pact createFromFeignClient(Class<?> feignClient, String consumerName, Obj
         return Pact.builder()
             .provider(new Service(extractProviderName(feignClient)))
             .consumer(new Service(consumerName))
-            .interactions(createInteractionsFromMethods(methodExtractor, feignClient.getMethods(), objectMapper))
+            .interactions(createInteractionsFromMethods(
+                methodExtractor, feignClient.getMethods(), objectMapper, extractPathPrefix(feignClient)))
             .metadata(new Metadata("1.0.0"))
             .build();
     }
 
+    private String extractPathPrefix(Class<?> feignClient) {
+        FeignClient feignClientAnnotation = feignClient.getAnnotation(FeignClient.class);
```

Comment:
no need for one-time var here, you can write `return feignClient.getAnnotation(FeignClient.class).path();`